tests/plugins/test_koji_promote.py

```diff
@@ -961,13 +961,15 @@ def test_koji_promote_image_tags(self, tmpdir, os_env, additional_tags):
          None),
 
     ])
-    @pytest.mark.parametrize('has_config', [
-        True,
-        False,
+    @pytest.mark.parametrize(('has_config', 'is_autorebuild'), [
+        (True,
```

Comment:
Each tuple in the list is a test case, and they are both the same, meaning it's the same test case twice. Please read the docs: http://doc.pytest.org/en/latest/parametrize.html